frontend/src/component/releases/ReleaseManagement/ReleaseManagement.tsx

```diff
@@ -10,12 +10,20 @@ import { useNavigate } from 'react-router-dom';
 import { useReleasePlanTemplates } from 'hooks/api/getters/useReleasePlanTemplates/useReleasePlanTemplates';
 import { EmptyTemplatesListMessage } from './EmptyTemplatesListMessage';
 import { ReleasePlanTemplateList } from './ReleasePlanTemplateList';
+import { useUiFlag } from 'hooks/useUiFlag';
+import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
 
 export const ReleaseManagement = () => {
     usePageTitle('Release management');
     const navigate = useNavigate();
     const data = useReleasePlanTemplates();
 
+    const { isEnterprise } = useUiConfig();
+    const releasePlansEnabled = useUiFlag('releasePlans');
+    if (!releasePlansEnabled) {
+        return null;
+    }
```

Comment:
I think this is fine if the feature is disabled you should not be able to see this... I don't think it's needed due to https://github.com/Unleash/unleash/blob/3696420c51dc9d673c6da1393c4d0a61b8c3af24/frontend/src/component/menu/routes.ts#L290-L292 but :shrug: